packages/smock/src/smockit/smockit.ts

```diff
@@ -165,13 +165,11 @@ export const smockit = async (
   // We attach a wallet to the contract so that users can send transactions *from* a smock.
   await hre.network.provider.request({
     method: 'hardhat_impersonateAccount',
-    params: [contract.address]
+    params: [contract.address],
```

Comment:
Whoops, look like CI isn't properly checking for linting errors in the Smock package. Leaving a comment here to remind myself to fix this on Monday.

Reply:
(Or someone else can fix if you want!)